frontend/src/metabase/visualizations/components/Visualization/Visualization-table.unit.spec.js

```diff
@@ -50,10 +50,15 @@ describe("Table", () => {
 
     const bgColors = rows
       .map(([value]) => screen.getByText(String(value)))
-      .map(element => element.parentNode.style["background-color"]);
+      .map(element =>
+        window
+          .getComputedStyle(element.parentNode)
+          .getPropertyValue("background"),
+      );
+
     expect(bgColors).toEqual([
-      "",
-      "",
+      "rgb(255, 255, 255)",
```

Comment:
Just to be safe as [mentioned in this comment.](https://github.com/metabase/metabase/pull/42411#discussion_r1599588668) I'd refrain from changing the default behavior outside SDK context. Although, it's possible that the implementation changes, but the behavior hasn't, but it could take time to test this.

Reply:
@WiNloSt I believe this is because the test was asserting `style.backgroundColor` instead of getting the actual computed styles. I checked that the CSS class names makes the color to be `rgb(255, 255, 255)`, but the tests was asserting only inline styles - so when we change this to `getComputedStyles` it picked up the color from the class names.

Reply:
It's a good reminder to check the behaviour for `theme=transparent` though, I'll keep that in mind when I test.

Reply:
Oh that sounds good. Thanks for the explanation.

Reply:
You're right actually, just noticed that it's supposed to be unset - not white - from the failing unit tests. I reverted the above change and now it passes.